cosmo_tester/framework/cluster.py

```diff
@@ -275,15 +275,24 @@ def _get_latest_manager_image_name():
     For CLI version "4.0.0-m15"
     Returns: "cloudify-manager-premium-4.0m15"
     """
-    version = util.get_cli_version()
-    version_num, _, version_milestone = version.partition('-')
+    specific_manager_name = ATTRIBUTES.cloudify_manager_latest_image.strip()
 
-    if version_num.endswith('.0') and version_num.count('.') > 1:
-        version_num = version_num[:-2]
+    if specific_manager_name:
+        image_name = specific_manager_name
```

Comment:
I was actually trying to follow the 'only one return' approach, partially for readability (I think it makes it clearer when you can follow one branch or the other- immediately visually obvious that the other branch only gets executed when <condition> is false), and partially because if we need to do any post-processing on the image later (or logging, e.g. of image UUID) then it becomes easier to.

Of course, the second point may never be needed.

How strongly do you feel either way about it? I don't think we have a strong style preference on this in the code base?